### ISSUE TITLE
supporting previous sdk version

### ISSUE THREAD

**Issue by touseefbsb** [Microsoft/WindowsTemplateStudio#2092]
Currently WTS generates projects with creators update as minimum version, but if I generate a project and then set the minimum version to Anniversary update because I want to support that version as well, will that work? or I might get any exception in a certain scenario?

**Comment by touseefbsb** [comment]
ok got it :)

**Comment by touseefbsb** [comment]
@mrlacey  I created the project with the latest version and put its min and target version both to **Anniversary Update** and now I get following build error

**Cannot find type Windows.UI.Xaml.Media.XamlCompositionBrushBase in module Windows.Foundation.UniversalApiContract.winmd**

I knw it is caused by only version change because it can build with min = creators and target = FCU.

If you have any idea how I can resolve this issue that would be really helpful. I didnt make a seperate issue for this because I know wts cannot take responsibility for previous sdks. I have removed all the conditional xaml on all pages and my app is completely adapted for anniversary update, if only I can figure out root cause of this build error I can make this project work properly on anniversary update .
